sale_order_warn_message/models/sale_order.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    sale_warn_msg = fields.Text(compute="_compute_sale_warn_msg")
```

Comment:
A remark about this is the field is stored in sale module. A search method could be necessary